server/routes/induction/update/qualificationsListUpdateController.ts

```diff
@@ -126,3 +128,13 @@ const inductionWithRemovedQualification = (
     },
   }
 }
+
+const updatedInductionDtoWithDefaultedHighestLevelOfEducation = (inductionDto: InductionDto): InductionDto => {
```

Comment:
There is a horrible edge case with the current Induction question sets
If you originally do a short question set, you are asked if you want to add qualifications, but not what your highest level of education was. It's simply not asked! 🤷‍♂️ 
The long question set does answer it though.

So, in the case of what we are doing here - updating a short question set to a long question set - you can arrive on the educational qualifications screens with some missing data, specifically highest level of education.

One of the options on the highest level of education screens is "Not sure", so our safest option is to default it to Not Sure if we dont have the answer.

It needs to be handled like this, but it is a real edge case. It only happens if the original Induction was a short question set, AND they choose to add at least qualification (they dont need to in the short question set); AND they then change the "do you want to work" to YES which then kicks off this long question set journey